dnSpy/dnSpy/MainApp/AppCommandLineArgs.cs

```diff
@@ -262,5 +269,62 @@ public AppCommandLineArgs()
 		}
 
 		public IEnumerable<Tuple<string, string>> GetArguments() => userArgs.Select(a => Tuple.Create(a.Key, a.Value));
+
+
+
+		public void ShowHelp() {
+
+
+			var text = GetHelpText();
+			if (GetConsoleWindow() == IntPtr.Zero) {
+				// AttachConsole(/*ParentProcess*/-1) didn't work here
+				// Console.WriteLine works, but the ScreenBuffer of a real Console will not be updated.
+
+				// Use a native msgbox instead (MsgBoxService is not set at this time)
+				System.Windows.MessageBox.Show(text, "dnSpy Commandline help");
+			} else {
+				Console.WriteLine(text);
+			}
+		}
+
+		public string GetHelpText() {
+			// USE SPACES!!
```

Comment:
Max row size isn't needed when it's shown in a dialog box so you can join some of the lines.